src/s3/commands/downloadFileAs.ts

```diff
@@ -3,23 +3,25 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { extname } from 'path'
+import { extname, join } from 'path'
```

Comment:
Nit: we usually do `import * as path from 'path'`. I don't think we have a hard-and-fast reason why but something like `path.join()` makes more logical sense to me than `join()` does.

Reply:
Makes sense